doc/modules/mod_inbox.md

```diff
@@ -0,0 +1,74 @@
+### Module Description
+
+It's an experimental XMPP inbox feature.
+This extension consists of several modules but only `mod_inbox` needs to be enabled in the config file.
+
```

Comment:
Doc should describe that this extension depends on `mod_mam` (with proper configuration in order to work with groupchats).

Reply:
not depending on MAM anymore